features/netsocket/LoRaRadio.h

```diff
@@ -0,0 +1,402 @@
+/**
+ * Copyright (c) 2017, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef LORARADIO_H_
+#define LORARADIO_H_
+
+// Radio wake-up time from sleep - unit ms.
+#define RADIO_WAKEUP_TIME                           1
```

Comment:
Who uses this and why not part of the local phy scope?

Reply:
This is a left over, I will clean it.

Reply:
Okay, I studied it a bit more. What I find is that this time is used for calculating rx windows parameters and is used by mac and all phy layers. Could move it to lorawan_data_structures though